scalameta/scalameta/shared/src/test/scala/scala/meta/tests/parsers/ModSuite.scala

```diff
@@ -367,14 +367,12 @@ class ModSuite extends ParseSuite {
 
   test("invalid private and protected") {
     interceptParseErrors(
-      "private protected class A",
-      "protected private class A",
-      "private[something] protected class A",
-      "protected private[something] class A",
-      "protected[something] private class A",
-      "private protected[something] class A",
```

Comment:
Turns out that `private[foo] protected` and friends are actually valid Scala code. We actually encountered that in our codebase.